### DISCUSSION TITLE
i need change this configuration. then how to change?

### DISCUSSION THREAD

**Discussion by omkarade** [coqui-ai/TTS#1741] [General Q&A]
Preprocessing samples
 | > Max text length: 3
 | > Min text length: 3
 | > Avg text length: 3.0
 | 
 | > Max audio length: 1179556.0
 | > Min audio length: 264880.0
 | > Avg audio length: 417389.8004158004

**Comment by erogol** [comment]
Do not spam the discussions with the same questions, please. Someone will help at his/her own pace.
